lib/parity/environment.rb

```diff
@@ -128,6 +128,14 @@ def basename
     end
 
     def run_migration?
+      a_rails_app? && pending_migrations?
+    end
+
+    def a_rails_app?
```

Comment:
Nice. Hound already recommended one change from `is_a_rails_app?`, so now we're getting shorter AND more succinct.